bodhi/server/config.py

```diff
@@ -407,6 +407,12 @@ class BodhiConfig(dict):
         'initial_bug_msg': {
             'value': '%s has been submitted as an update to %s. %s',
             'validator': unicode},
+        'greenwave_api_url': {
+            'value': 'https://greenwave.fedoraproject.org/api/v1.0',
+            'validator': unicode},
+        'greenwave_enabled': {
+            'value': False,
+            'validator': _validate_bool},
```

Comment:
@matt8754 I'm fine with keeping the existing code that blocks on resultsdb results if you like - I think that's a fine reason.

I like ```test_gating.enabled```. Bodhi in general does gating, so a setting that just says "gating" could sound like it's disabling all gating. How about we rename ```ci.enabled``` to ```test_gating.enabled``` and use that everywhere?